ansible/roles/mn-evo-services/templates/tendermint/genesis.json.j2

```diff
@@ -1,43 +1,3 @@
 {
-  "genesis_time": "{{ genesis_time }}",
-  "chain_id": "dash-{{ ( 'devnet-' + dash_devnet_name if dash_network == 'devnet' else dash_network ) if tenderdash_chain_id is not defined else tenderdash_chain_id }}",
-  "initial_height": "0",
```

Comment:
I would keep `genesis_time`, `initial_height`, `initial_core_chain_locked_height`, `consensus_params`, and `quorum_type` to make it more verbose and reduce future confusions.

@lklimek shall we rename:
- `threshold_public_key` -> `validator_quorum_threshold_public_key`
- `quorum_type` -> `validator_quorum_type`
- `quorum_hash` -> `validator_quorum_hash`